CHANGELOG.md

```diff
@@ -2,13 +2,19 @@
 
 All notable changes to this project will be documented in this file.
 
-## [Unreleased]
+See Annotation for Clinical Data [release notes](https://cloud.ibm.com/docs/wh-acd?topic=wh-acd-release-notes) for latest features.
 
-## [0.0.1] - 2019-02-15
+## [Released]
+
+## v2.0 - build v2.0.202106231922
+
+### General Availability
```

Comment:
Maybe drop the heading here and just make this text? Would have to see how that looks when formatted that way, and change to "General Availability of IBM Watson Annotator for Clinical Data Container Edition".